src/custom/theme/cowSwapTheme.tsx

```diff
@@ -162,7 +195,7 @@ export function theme(darkmode: boolean): DefaultTheme {
 
     // Overide Theme
     ...baseThemeVariables(darkmode, colorsTheme),
-    ...themeVariables(colorsTheme)
+    ...themeVariables(darkmode, colorsTheme)
```

Comment:
switch the param order around if you're going to do this to keep it as similar as Uni's

Reply:
Not sure I understand: Didn't we already have this order in place, looking at `...baseThemeVariables(darkmode, colorsTheme),` ? Didn't see direct examples on the original UNI code.

Reply:
theme Variables is our code? Then all good,, thought it was a function coming from Uni we extended